### ISSUE TITLE
Invalid parameter?

### ISSUE THREAD

**Issue by Gruntfuggly** [lannonbr/vscode-js-annotations#23]
Just thought I'd try out your extension after reading your decorations article. 8-)

I see a few 'invalid parameters' where I wouldn't expect them, e.g.

![image](https://user-images.githubusercontent.com/376658/45284628-f029b180-b4d8-11e8-9dd9-f8aa51a3ba41.png)

Is this a limitation of what can be detected? Apart from that, it looks really useful - thanks!

**Comment by lannonbr** [comment]
Looking at the function definition when you hover over that function, it should be able to see the two parameters and annotate that function call correctly so I will look into trying to replicate this scenario and then see if it can be resolved.

**Comment by lannonbr** [comment]
I was not able to duplicate the issue with a simple example.

Could you put your cursor on getElement and either hit F12 or run "Go to definition" and make sure whatever it returns has 2 parameters.

As well, @Gruntfuggly Would it be possible if I could have a snippet of the code which breaks or some instructions on trying to reproduce this bug?

Thanks!

**Comment by Gruntfuggly** [comment]
I can get it to happen with these two files:

`extension.js`
```
var MyView = require( "./dataProvider" );
var provider = new MyView.DataProvider();
var filename = "";
provider.getElement( "", filename );
```

`dataProvider.js`
```
class DataProvider
{
    getElement( folder, filename ) {}
}
exports.DataProvider = DataProvider;
```

If that doesn't work, let me know - maybe it's another extension interfering?

**Comment by lannonbr** [comment]
Found out the issue. It's because there are spaces in the definition of getElement in dataProvider.js. I did not account for that it seems but I will work on getting a patch shortly to fix it

**Comment by lannonbr** [comment]
Resolved in 0.7.3
